src/styles/_variables.scss

```diff
@@ -0,0 +1,18 @@
+$sp-1: 0.063rem;
+$sp-5: 0.313rem;
+$sp-8: 0.5rem;
+$sp-14: 0.875rem;
+$sp-20: 1.25rem;
+$sp-24: 1.5rem;
+$sp-26: 1.625rem;
+$sp-36: 2.25rem;
+$sp-56: 3.5rem;
+$sp-70: 4.375rem;
+$sp-72: 4.5rem;
+$sp-76: 4.75rem;
+$sp-96: 6rem;
+$sp-103: 6.438rem;
+$sp-112: 7rem;
+$sp-123: 7.688rem;
+$sp-150: 9.375rem;
+$sp-153: 9.563rem;
```

Comment:
use 2, 4, 8, 12, 16.... sequence. Knock me if you have any confusion.

Reply:
I have used 2,4,8,12,20,24,36,48